README.md

```diff
@@ -145,16 +169,15 @@ code, include a script or makefile that will do the job.
 
 ## A2.5
 
-«replace this with your answer»
-
+fsm.java
```

Comment:
# 10

It's a pretty implementation, but the question asked for the table to be expressed in your language: I was expecting it to be in data, not code.